src/components/src/test/groovy/org/apache/jmeter/extractor/json/render/RenderAsJsonRendererSpec.groovy

```diff
@@ -112,7 +108,7 @@ class RenderAsJsonRendererSpec extends JMeterSpec {
     def "setupTabPane adds the tab to rightSide"() {
         given:
             sut.init()
-            def rightSideTabbedPane = new JTabbedPane();
+            def rightSideTabbedPane = new JTabbedPane()
```

Comment:
I wish this "optional semicolon" was represented in Spotless somehow.

Reply:
Me, too, maybe a suggestion for them?

Reply:
See #518